nextcord/member.py

```diff
@@ -718,10 +721,22 @@ async def edit(
             Pass ``None`` to kick them from voice.
         reason: Optional[:class:`str`]
             The reason for editing this member. Shows up on the audit log.
-        timeout: Optional[:class:`datetime.datetime`]
+        timeout: Optional[Union[:class:`~datetime.datetime`, :class:`~datetime.timedelta`]
             The time until the member should not be timed out.
             Set this to None to disable their timeout.
 
+            Example
```

Comment:
Not sure if having a example inside a arg is a good idea... How does this look on the docs?

Reply:
It looks alright actually